Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/directives/localizeOptions.Directive.js

```diff
@@ -0,0 +1,12 @@
+﻿angular.module("umbraco.directives").directive('localizeOption', function (localizationService) {
+    return {
+        restrict: 'A',
+        link: function ($scope, element, attrs, controller) {
+            var key = attrs.localizeOption;
+
+            localizationService.localize(key).then(function (value) {
+                element.text(value);
+            });
+        }
+    }
+});
```

Comment:
@bjarnef this looks neat & wraps Umbraco's localizationService yes?
If so then surely Umbraco has their own directive that would do this already in the core no?